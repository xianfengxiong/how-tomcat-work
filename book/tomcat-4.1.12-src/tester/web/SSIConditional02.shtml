Before if block.
<!--#set var="test" value="value2" -->
<!--#if expr="\"$test\" = \"value1\"" -->
test = value1
<!--#elif expr="\"$test\" = \"value2\"" -->
test = value2
<!--#else -->
test = neither value1 or value2
<!--#endif -->
After if block.
