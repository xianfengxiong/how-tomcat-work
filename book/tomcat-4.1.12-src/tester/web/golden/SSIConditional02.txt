Before if block.


test = value2

After if block.
